server/game/cards/01-DTR/PointBlank.js

```diff
@@ -0,0 +1,37 @@
+const ActionCard = require('../../actioncard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class PointBlank extends ActionCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Resolution: Point Blank',
+            playType: 'resolution',
+            choosePlayer: false,
+            cost: [
+                ability.costs.boot(card =>
+                    card.getType() === 'dude' &&
+                    card.isStud() && 
+                    card.isParticipating()
+                )
+            ],
+            handler: context => {
+                this.game.promptForSelect(context.player.getOpponent(), {
+                    activePromptTitle: 'Select a dude to ace',
+                    waitingPromptTitle: 'Waiting for opponent to select dude',
+                    cardCondition: { location: 'play area', controller: context.player.getOpponent(), participating: true, condition: card => card.bullets < context.costs.boot.bullets },
```

Comment:
This could be done in 2 ways and you selected one of them so no problem here. Just FYI the second would be to pass `context` to properties of `promptForSelect` and then using `controller: 'opponent'` in cardCondition.
It would look like this:
```
// ...
waitingPromptTitle: 'Waiting for opponent to select dude',
context: context,
cardCondition: { location: 'play area', controller: 'opponent', participating: true, condition: card => card.bullets < context.costs.boot.bullets },
// ...
```